src/libs/MoneyRequestUtils.ts

```diff
@@ -15,6 +15,13 @@ function stripSpacesFromAmount(amount: string): string {
     return amount.replace(/\s+/g, '');
 }
 
+/**
+ * Strip decimals from the amount
+ */
+function stripDecimalsFromAmount(amount: string): string {
+    return amount.replace(/\.\d*/, '');
```

Comment:
Yeah, instead of matching anywhere in the string, it may be better to explicitly match it till the end 👍 (not sure if this would change anything because we already have validation to prevent multiple decimal dots).